File: Python/PRAK104-2210817220018-Piola Evania.py
SA = 400000
SB = 350000
DSA = 0.13
DSB = 0.21
HSA = SA-DSA*SA
HSB = SB-DSB*SB

print("Harga sepatu A adalah ", SA)
print("Harga sepatu B adalah ", SB)
print("Sepatu A mendapat diskon 13% sehingga harganya menjadi ", round(HSA))
print("Sepatu B mendapat diskon 21% sehingga harganya menjadi ", round(HSB))

File: Modul 2/Soal 4/PRAK104-2210817220018-Piola Evania.c
#include <stdio.h>

int main()
{
    float SA = 400000;
    float SB = 350000;
    float DSA = 13;
    float DSB = 21;
    float HSA = SA-DSA/100*SA;
    float HSB = SB-DSB/100*SB;
    char D = '%';

    printf("Harga sepatu A adalah %.f\n", SA);
    printf("Harga sepatu B adalah %.f\n", SB);
    printf("Sepatu A mendapat diskon 13%c sehingga harganya menjadi %.f\n", D, HSA);
    printf("Sepatu B mendapat diskon 21%c sehingga harganya menjadi %.f\n", D, HSB);
    return 0;
}
